.github/workflows/pytest.yaml

```diff
@@ -24,7 +24,7 @@ jobs:
         run: poetry build
 
       - name: Checkout test environment
-        run: git clone https://github.com/exasol/integration-test-docker-environment.git
+        run: git clone -b 1.6.0 https://github.com/exasol/integration-test-docker-environment.git
         working-directory: ..
```

Comment:
You could also use `actions/checkout` ;)

Reply:
this repo gets archived soon, so not worth it